### DISCUSSION TITLE
windows配置环境出现了could not find 'Cargo.toml'错误

### DISCUSSION THREAD

**Discussion by cl8600** [NonceGeek/Move-Camp#221] [Q&A] [answered]
各位大佬们还有maintainer，已经捋出来了问题：
我下载的aptos-cli是aptos-cli-1.0.3-Ubuntu-x86_64版本，里面只有一个aptos文件。通过远程桌面连接，发现这个文件在/home/cl8600/foo/bin中。
![图片](https://user-images.githubusercontent.com/96363155/210783671-fd834037-47f5-42fc-b0d7-ec10a74be61a.png)
但是运行命令cargo build --release
cp home/cl8600/foo/bin/aptos 的时候报了could not find 'Cargo.toml'文件这个错误。
![图片](https://user-images.githubusercontent.com/96363155/210783742-dbe4ec36-423c-4d1d-b65e-35432bab5a9e.png)
请求解决办法😁

**Comment by cl8600** [comment]
这个是在window中用vscode+wsl安装的环境出现的问题

**Comment by v1xingyue** [comment, marked as answer]
强烈建议开发别在window 里搞了，会有各种你想不到的坑。

如果实在不想抛弃当前系统，弄个虚拟机吧 VMware 或者别的虚拟机软件都行。

如果觉得里边的界面不爽，可以尝试 vscode remote开发。

再或者，不想搭环境，用 play ground 吧 

https://playground.pontem.network/
